extensions/amp-video-docking/0.1/test/test-amp-video-docking.js

```diff
@@ -174,6 +175,35 @@ describes.repeated('', {
       viewport.height = 0;
     });
 
+    describe('#getPosterImageSrc_', () => {
+
+      skipForSlot('uses `poster` attr', () => {
+        const html = htmlFor(env.win.document);
+        const el = html`<amp-video poster=foo.png></amp-video>`;
+
+        expect(docking.getPosterImageSrc_(el)).to.equal('foo.png');
+      });
+
+      skipForSlot('uses `placeholder` amp-img', () => {
+        const html = htmlFor(env.win.document);
+        const el = html`<amp-video>
+          <amp-img src=foo.png placeholder></amp-img>
+        </amp-video>`;
+
+        expect(docking.getPosterImageSrc_(el)).to.equal('foo.png');
+      });
+
+      skipForSlot('uses `placeholder` img', () => {
+        const html = htmlFor(env.win.document);
+        const el = html`<amp-video>
+          <img src=foo.png placeholder>
```

Comment:
ditto above, I didn't think this is valid.

Reply:
Not valid AMPHTML but generated by the runtime.